.rubocop.yml

```diff
@@ -1,8 +1,109 @@
-inherit_from:
-  - https://raw.githubusercontent.com/carbonfive/c5-conventions/master/rubocop/rubocop.yml
+require:
+  - rubocop-performance
 
 AllCops:
+  DisplayCopNames: true
+  DisplayStyleGuide: true
   TargetRubyVersion: 2.4
```

Comment:
My understanding is that rubocop automatically detects the ruby version and this line can be removed.

Reply:
If I remove it I get this error:

```
socrates.gemspec:25:32: C: Gemspec/RequiredRubyVersion: required_ruby_version (2.4, declared in socrates.gemspec) and TargetRubyVersion (2.7, which may be specified in .rubocop.yml) should be equal.
  spec.required_ruby_version = ">= 2.4.0"
                               ^^^^^^^^^^
```

🤷‍♂